sbncode/FlashMatch/FlashPredict_module.cc

```diff
@@ -1336,12 +1394,13 @@ FlashPredict::ChargeDigestMap FlashPredict::makeChargeDigest(
         flashmatch::QCluster_t hitsClusters;
         std::set<unsigned> hitsTPCs;
         for(const auto& hit : hits) {
+          int hit_plane = (int)hit->View();
           geo::WireID wId = hit->WireID();
-          if(fOnlyCollectionWires &&
-             fGeometry->SignalType(wId) != geo::kCollection) continue;
+          if(std::find(fPlaneList.begin(), fPlaneList.end(), hit_plane) == fPlaneList.end()) continue;
           const double hitQ = hit->Integral();
-          if(hitQ < fMinHitQ) continue;
-          spacepointQ += hitQ;
+          if(!fAllPlanes) {
```

Comment:
This check should be made in the previous, to skip comparing the elements of `fPlaneList` if `fAllPlanes` is true.